sgm/modules/autoencoding/losses/__init__.py

```diff
@@ -3,10 +3,9 @@
 import torch
 import torch.nn as nn
 from einops import rearrange
-from taming.modules.discriminator.model import NLayerDiscriminator, weights_init
-from taming.modules.losses.lpips import LPIPS
-from taming.modules.losses.vqperceptual import hinge_d_loss, vanilla_d_loss
-
+from ...diffusionmodules.lpips.lpips import LPIPS
```

Comment:
I would rather place the `lpips` directory in `sgm/modules/autoending` since autoencoder training is where nearly all the classes in LPIPS are actually used

Reply:
Can do that

Reply:
Done